docs/spec_v6.md

```diff
@@ -0,0 +1,220 @@
+---
+id: "spec_v6"
+title: "Twirp Wire Protocol"
+sidebar_label: "Version 6 (Proposed Draft)"
+---
+
+This document is a draft of a new version ("v6") of the Twirp wire
+protocol over HTTP. This draft is EXPERIMENTAL and subject to change.
+v6 of Twirp is not released or considered stable; instead, this
```

Comment:
"not yet released ...", unless you want something like "even-numbered releases are unstable"

Reply:
Sure, not yet released - I'm just trying to explain that it's a draft.